incompatibility/incompatibility-report/incomp-scanning_test.go

```diff
@@ -45,16 +47,49 @@ func TestBasicSecurityIncompatibility(t *testing.T) {
 	path1 := "../../generator/testfiles/other.yaml"
```

Comment:
declare these inline with the struct to prevent accidentally usage.

Reply:
fixed